### ISSUE TITLE
ActionBarSherlock is always required dependency

### ISSUE THREAD

**Issue by futtetennista** [saik0/UnifiedPreference#5]
Hi Joel,

Thanks very much for this library, it's extremely useful and it works just fine. One issue I had though is that it always requires ActionBarSherlock as a dependency and I don't think one should be forced to use it - even if one could argue that is not such a wise choice. I worked on removing ABS and adding support to it in a separate project, taking the same approach Jake Wharton took with the ABS Map plugin. You can check these two links out and tell me what you think about it, if you think it's fine I'll open a pull request: https://github.com/stefanodacchille/UnifiedPreference/tree/plugin-actionbarsherlock, and https://github.com/stefanodacchille/UnifiedPreference-ActionBarSherlockPlugin .

**Comment by saik0** [comment]
Nice work. I will cherry-pick from your repo. I do not want to change attr names.

**Comment by futtetennista** [comment]
Thanks. Not sure it'll work without changing those attribute name though, I remember trying to do that and failing. But maybe I was just missing something.

**Comment by saik0** [comment]
Ah, I see what's happening with those duplicate attrs now. Very frustrating.

**Comment by saik0** [comment]
> Thanks. Not sure it'll work without changing those attribute name though, I remember trying to do that and failing. But maybe I was just missing something.

Library projects don't have their own attribute namespace, they get merged into your project. Before we were using the title and icon attrs defined in ABS, note how the xml nodes had no `format` attribute. Hopefully the "new build system" will have proper libraries, but I suspect it wont be released until after ABS and this project are not needed.

**Comment by futtetennista** [comment]
Of course, you're right. Then unfortunately those must change. It sucks
that is a breaking change but I think it's still a cleaner approach.

**Comment by saik0** [comment]
Did you manage to get the demo to build with maven? I get this error:

UnifiedPreference-ActionBarSherlockPlugin/demo/AndroidManifest.xml:28: error: Error: No resource found that matches the given name (at 'theme' with value style/Theme.Sherlock').

**Comment by futtetennista** [comment]
Uhm, no I can build the project without errors. I fixed a crash at Runtime
when using Maven to build the app though: I forgot to set the correct
source directory in plugin and demo POM files.
